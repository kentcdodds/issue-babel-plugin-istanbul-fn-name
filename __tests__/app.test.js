const App = require('../app')
const React = require('react')
const toJSON = require('enzyme-to-json').default
const shallow = require('enzyme').shallow

describe('app', () => {
  it('should render', () => {
    const tree = toJSON(shallow(<App />))

    expect(tree).toMatchSnapshot()
  })
})
