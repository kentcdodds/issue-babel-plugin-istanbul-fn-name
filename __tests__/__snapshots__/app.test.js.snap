exports[`app should render 1`] = `
<div>
  <Header />
  <main>
    I am content
  </main>
  <Footer />
</div>
`;
